recipes/r-rserve/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
you mean setting `export R_HOME=$PREFIX/lib/R`?  Shouldn't `rpaths: - lib/R/lib/` in meta.yaml do exactly that?